### PR TITLE
On firefox scroll fixed

### PR DESCRIPTION
**Pull request by Aca-jov**
SD-3927, SD-3922, SD-3933

### CODE REVIEW

File: client/app/scripts/superdesk-monitoring/monitoring.js [hunk 1]
```diff
@@ -546,7 +546,6 @@
                                 list.style.height = (scope.total * ITEM_HEIGHT) + 'px';
                             }
 
-                            list.style.paddingTop = (from * ITEM_HEIGHT) + 'px';
```
Comment (Aca-jov) [inline client/app/scripts/superdesk-monitoring/monitoring.js]: @petrjasek Can you please check this, you worked on it. This padding top was problem, but I'm not sure why is this padding for, maybe I broke something somewhere else..

Comment (petrjasek) [inline client/app/scripts/superdesk-monitoring/monitoring.js]: hey, I think it's needed.. if you have longer list and you scroll further down this should position the items properly. without having the scroll bar jumping.

Comment (Aca-jov) [inline client/app/scripts/superdesk-monitoring/monitoring.js]: Ah, ok, I didn't notice that. I fixed it with that, so now it's fine. Thanks

**Comment by pavlovicnemanja** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**